### DISCUSSION TITLE
Welcome to vorp_inventory-lua Discussions!

### DISCUSSION THREAD

**Discussion by outsider31000** [VORPCORE/vorp_inventory#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cassieboon** [comment]
hi how do i change the inventory size from 35 to what ever i want it to be? ive searched all the lua's and cant find it anywhere
